src/common/zip.js

```diff
@@ -1,18 +1,8 @@
 import { memoize } from './util';
 
-function loadScript(url) {
-  return new Promise((resolve, reject) => {
-    const script = document.createElement('script');
-    script.src = url;
-    script.onload = () => resolve();
-    script.onerror = () => reject();
-    document.body.append(script);
-  });
-}
-
 const loadZip = memoize(async () => {
-  await loadScript('/public/lib/zip.js/zip.js');
-  return window.zip;
+  const zip = await import(/* webpackChunkName: 'zip' */ '@zip.js/zip.js/dist/zip');
```

Comment:
Neat! But it creates `vendors~zip.js` in the root directory so maybe add `...splitVendor('@zip'),` in plaid.conf.js?

Reply:
Done!